engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/result/NadelResultTransformer.kt

```diff
@@ -74,7 +74,8 @@ internal class NadelResultTransformer(private val executionBlueprint: NadelOvera
             it.run(context = transformContext)
         }
 
-        transformContext.emptyPaths.forEach(result.data::cleanup)
+        // TODO: investigate whether this code is actually needed. So far no tests fail after disabling it…
+        // transformContext.emptyPaths.forEach(result.data::cleanup)
```

Comment:
So this code nulls out a `Map` that has been edited in the transformation process _if_ it is empty. I guess nobody actually does that.